packages/react-core/src/demos/Chart/PrometheusChart.md

```diff
@@ -0,0 +1,207 @@
+import HelpIcon from '@patternfly/react-icons/dist/js/icons/help-icon';
+import imgBrand from '@patternfly/react-core/src/components/Brand/examples/pfLogo.svg';
+import imgAvatar from '@patternfly/react-core/src/components/Avatar/examples/avatarImg.svg';
+import mockData from './mockData.js';
+
+PrometheusChart = () => {
+  const [activeItem, setActiveItem] = React.useState(0);
+  const [width, setWidth] = React.useState(0);
+  const [isStacked, setIsStacked] = React.useState(true);
+  const containerRef = React.useRef(null);
+  const legendData = [];
+  const tooltipSeriesNames = [];
+  const results = mockData.data.result;
+
+  const handleResize = () => {
+    if (containerRef.current && containerRef.current.clientWidth) {
+      setWidth(containerRef.current.clientWidth);
+    }
+  };
+
+  React.useEffect(() => {
+    handleResize();
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const timeFormatter = new Intl.DateTimeFormat('en', {
+    hour: 'numeric',
+    minute: 'numeric'
+  });
+
+  const dateTimeFormatterWithSeconds = new Intl.DateTimeFormat('en', {
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    year: 'numeric'
+  });
+
+  const formatSeriesValues = values => {
+    const newValues = values.map(v => {
+      const y = Number(v[1]);
+      return {
+        x: new Date(v[0] * 1000),
+        y: Number.isNaN(y) ? null : y
+      };
+    });
+    return newValues;
+  };
+
+  const data = results.map(({ metric, values }) => {
+    return [metric, formatSeriesValues(values)];
+  });
+
+  data.forEach(([metric, values], i) => {
+    const name = `cpu${metric.cpu}`;
+    legendData.push({ name, childName: metric.cpu });
+    tooltipSeriesNames.push(name);
+  });
+
+  const CursorVoronoiContainer = createContainer('voronoi', 'cursor');
+
+  const graphContainer = (
+    <CursorVoronoiContainer
+      cursorDimension="x"
+      labels={({ datum }) => `${datum.y !== null ? datum.y : 'no data'}`}
+      labelComponent={
+        <ChartLegendTooltip legendData={legendData} title={datum => dateTimeFormatterWithSeconds.format(datum.x)} />
+      }
+      mouseFollowTooltips
+      voronoiDimension="x"
+      voronoiPadding={50}
+    />
+  );
+
+  const theme = getCustomTheme(ChartThemeColor.multiUnordered, ChartThemeVariant.light);
+  const colors = theme.line.colorScale;
+
+  const xAxisTickFormat = d => {
+    return timeFormatter.format(d);
+  };
+
+  const GroupComponent = isStacked ? ChartStack : ChartGroup;
+  const ChartComponent = isStacked ? ChartArea : ChartLine;
+
+  const PageToolbar = <div>need to implement new toolbar</div>;
+  const Header = (
+    <PageHeader
+      logo={<Brand src={imgBrand} alt="Patternfly Logo" />}
+      toolbar={PageToolbar}
+      avatar={<Avatar src={imgAvatar} alt="Avatar image" />}
+    />
+  );
```

Comment:
Not sure if we need to include the PatternFly avatar in the page header? The other demos don't seem to use it?